make/lint.mk

```diff
@@ -0,0 +1,9 @@
+# Lint targets for MetalK8S environment
+# Requirements for linting on host:
+#    - python3
+#    - tox
+
+lint-local:
```

Comment:
Just wondering, do we expect another kind of `lint-<something>` target? Maybe `lint` could be enough

Reply:
Originally I wanted to have an `rpmlint` run included, so my idea was to provide a target assuming dependencies first, then a target running this in a docker so a user wouldn't have to install `rpmlint`. I'll rename it for now, if we revisit that `rpmlint` stuff we can update this at that time.